platform/onebox/backup_and_restore/ceph/get-credentials.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+: '
+This scipt will generate json file for creds to be used with import export
+Script will generate file storage-creds.json
+Run it from the VM running aifabric.
+Use it as it is [insecure] or transfer to some credsManager and then change backup/restore scripts to fetch from credsmanager instead of json file
+[Script Version -> 21.4]
+'
+function initialize_variables() {
+	OBJECT_GATEWAY_INTERNAL_HOST=$(dig +short myip.opendns.com @resolver1.opendns.com)
```

Comment:
I think you are trying to find public IP of the machine here, what if machine doesn't have public IP and what if it's an airgapped machine.. I doubt dns lookup would work on airgap environment 

and it's not internatl host and internal port, they are external host and external port as the path is exposed via istio

Reply:
@rajivml since we planned to access the machine from another machine it will need an ip to directly access it. Even for air-gapped won't they have an ip, else how will ai-app work?

Reply:
Using hostname -i now